bokeh/server/core/server_document.py

```diff
@@ -0,0 +1,32 @@
+'''
+
+'''
+from __future__ import absolute_import
+
+from tornado import gen
+from tornado.queues import Queue
+
+class ServerDocument(object):
+
+    def __init__(self, docid):
+        self._docid = docid
+        self._queue = Queue
+        self._stopped = False
+
+    def stop_worker(self):
+        self._stopped = True
+
+    @gen.coroutine
+    def workon(self, item):
+        yield self._queue.put(item)
```

Comment:
This means you are not interested in the result of the processing, or even the information that the processing is done, right?